.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 /.build
+/.index-build
```

Comment:
why have you added this one?

Reply:
This is like a .build directory but for sourcekit-lsp.
In the sourcekit-lsp bundled with swift 6.1 it'll be moved into .build dir itself, but that's not out.

Reply:
you need to have Swift 6.0.x and have experimental-indexing enabled (e.g. in the VSCode extension or using the sourcekit-lsp config file).

Reply:
SwiftNIO already has it as well:
https://github.com/apple/swift-nio/blob/main/.gitignore#L3